applications/template/{{cookiecutter.project_slug}}/README.md

```diff
@@ -29,14 +29,38 @@ This application is built using Holoscan SDK version {{ cookiecutter.holoscan_ve
 ./holohub run {{ cookiecutter.project_slug }}
 ```
 
+### Running with Docker
```

Comment:
Can we update the "Running the Application" section above to clarify how to run containerized vs local?